lib/gateways/referral-gateway.test.js

```diff
@@ -349,8 +349,12 @@ describe('ReferralGateway', () => {
     });
   });
 
-  describe('save (actually update)', () => {
-    xit('saves the referral', async () => {
+  describe('update', () => {
+    it('updates the statusHistory for the referral', async () => {
+      const statusHistory = [
+        { status: 'SENT', date: '02/03/2020' },
```

Comment:
Should this `statusHistory` object contain `comments` field as well?

Reply:
it can contain comments, but for now it'll only be when the status is `rejected`
it will still allow us to add comments (and any other keys) in the future even if we don't specify it in the sent status